src/shared/seccomp-util.c

```diff
@@ -213,6 +231,84 @@ const SyscallFilterSet syscall_filter_sets[_SYSCALL_FILTER_SET_MAX] = {
                 "sigreturn\0"
                 "time\0"
         },
+        [SYSCALL_FILTER_SET_FILE_SYSTEM] = {
+                /* Basic file system access (reading file properties, opening/creating files and directories for
+                 * read+write) */
+                .name = "@file-system",
```

Comment:
@keszybz 
> Can we filter on open(…, O_CREATE) and mmap(…, PROT_WRITE) and move those to the latter group?

So the @basic-io group may make sense, however IMHO we should wait on `open(..., O_CREATE)` and writable operations... , and better figure out a setup where it is simple for users to use more simple switches and maybe embed the `open(O_CREATE)` inside without exposing it, or even better following these choices:
* ProtectSystem=strict
* RestrictMount=no|rdonly_devices|strict 

   strict: disables mount,umount,chroot and pivot_root
    rdonly_devices: if possible, allows chroot, and may imply some semantics of PrivateDevices. The idea is to not allow remounts, changing mount flags, ms_move etc on real devices, mount rw virtual filesystems can be ok, but we should prohibit write access to real devices, this way it will cover all the write syscalls on real hardware in one option, other stuff can be classified as memory...